frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-participants/experiment-participants.component.scss

```diff
@@ -91,6 +91,14 @@
       margin-left: 10%;
       width: 25%;
     }
+
```

Comment:
.no-include-message-container,
.no-exclude-message-container {
      color: #8F9BB3;
}

Let's combine these rules, it's same thing 😄 . Also, is this a standard color we are using elsewhere, particularly for this kind of message text? We should probably have a color variable defined globally for just about almost everything that's re-used (such as `variables.css`). Otherwise there are going to be inconsistent little color rules sprinkled everywhere.

Reply:
Yeah, I will combine them. And the color code is taken from the figma design. Okay, I will create a variable for. consistent color code for such messages.